mudslide/batch.py

```diff
@@ -17,6 +17,21 @@
 
 logger = logging.getLogger("mudslide")
 
+
+def boltzmann_velocities(mass, temperature, scale=True, seed=None): 
+    rng = np.random.default_rng(seed)
+    kt = boltzmann * temperature
+    sigma = np.sqrt(kt * mass)
+    p = rng.normal(0.0, sigma)
+    if scale:
+        avg_KE = 0.5 * np.dot(p**2, np.reciprocal(mass))/mass.size
+        kbT2 = 0.5 * kt
+        scal = np.sqrt(kbT2/avg_KE)
+        p *= scal
+    return p
```

Comment:
- I don't think this is the best place for this code. Maybe it'd be better in a new file for physical manipulations? Or `math.py`? Or `util.py`? I'm not sure, but it's out of place in `batch.py`
  - Add a test that covers this function.